fe/src/components/Login.vue

```diff
@@ -34,12 +35,17 @@ export default {
   },
   methods: {
     submitForm() {
-      // not sure what to do here 
-      var res = httpService.post('api/user/create/', {email:'test@email.ca', password:'password'});
-      console.log(res)
-      console.log(res.data)
 
-    }  	
+      this.$refs['LoginForm'].validate((valid) => {
+        if(valid){
+          alert('Logging in')
```

Comment:
If a user successfully logs in its a bit janky that they get a box they have to click through. 

consider https://element.eleme.io/#/en-US/component/notification

Reply:
Added some notifications to both pages if you take a look.